### ISSUE TITLE
SDL2 on Windows

### ISSUE THREAD

**Issue by CharlesBunders** [copperspice/cs_paint#4]
I am unable to get the CMake build to recognize that SDL2 is there.

From www.libsdl.org, I used the Runtime Binaries, the Development Libraries and just the Source Code setting "-Dsdl2_DIR=" to various directories which should have made it happy but to no avail.

cmake version 3.12.18081601-MSVC_2

**Comment by bgeller** [comment]
Have you looked at our documentation for building the CsPaint Demo?  We have information specific for Linux, Windows (MinGW), and MSVC.  This was updated last week with more detailed information about the environment variables.  

https://www.copperspice.com/docs/cs_paint/building-msvc.html

We just did a recent commit to adjust the Vulkan allocation (descriptor pool) and to add install rules for the CMake build files.  

For MSVC, the documented `sdl2-config.cmake` file needs one additional line added. Insert the following before before the first set :  `set(SDL2_PREFIX "${CMAKE_CURRENT_LIST_DIR}")`. 

On line documentation will be updated shortly. 

Barbara
